src/Node.js

```diff
@@ -0,0 +1,133 @@
+  // ### DOM Node Shim
+  //
+  // This represents a superficial form of a DOM node which contains most of
+  // the properties that a regular DOM node would contain, but only the ones
+  // needed by slowparse.
+module.exports = (function(){
+  "use strict";
+
+  function Node(nodeValue, namespaceURI) {
+    var self = this;
+    self.nodeValue = nodeValue || "";
+    self.namespaceURI = namespaceURI || "";
+    self.parentNode = false;
+    self.nextSibling = false;
+    self.isVoid = false;
```

Comment:
based on "don't do this in Document", this should just be `this.isVoid  = voidElements.indexOf(this.nodeName)>-1`